GCC warning for catch by value

Noticed a new warning with GCC 8+:
```
src/Xml/XmlAttribute.cpp: In member function ‘NAS2D::Xml::XmlAttribute::QueryResult NAS2D::Xml::XmlAttribute::queryIntValue(int&) const’:
src/Xml/XmlAttribute.cpp:196:14: warning: catching polymorphic type ‘class std::invalid_argument’ by value [-Wcatch-value=]
  catch (std::invalid_argument)
              ^~~~~~~~~~~~~~~~
src/Xml/XmlAttribute.cpp: In member function ‘NAS2D::Xml::XmlAttribute::QueryResult NAS2D::Xml::XmlAttribute::queryDoubleValue(double&) const’:
src/Xml/XmlAttribute.cpp:221:14: warning: catching polymorphic type ‘class std::invalid_argument’ by value [-Wcatch-value=]
  catch (std::invalid_argument)
              ^~~~~~~~~~~~~~~~
```

--- a/src/Xml/XmlAttribute.cpp
+++ b/src/Xml/XmlAttribute.cpp
@@ -193,7 +193,7 @@ XmlAttribute::QueryResult XmlAttribute::queryIntValue(int& i) const
 	{
 		i = std::stoi(_value);
 	}
-	catch (std::invalid_argument)
+	catch (const std::invalid_argument&)
 	{
 		return QueryResult::XML_WRONG_TYPE;
 	}
@@ -218,7 +218,7 @@ XmlAttribute::QueryResult XmlAttribute::queryDoubleValue(double& d) const
 	{
 		d = std::stod(_value);
 	}
-	catch (std::invalid_argument)
+	catch (const std::invalid_argument&)
 	{
 		return QueryResult::XML_WRONG_TYPE;
 	}

